gui-wm/hyprland/hyprland-9999.ebuild

```
@@ -0,0 +1,128 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit meson toolchain-funcs
+
+DESCRIPTION="A dynamic tiling Wayland compositor that doesn't sacrifice on its looks"
+HOMEPAGE="https://github.com/hyprwm/Hyprland"
+
+if [[ "${PV}" = *9999 ]]; then
+	EGIT_REPO_URI="https://github.com/hyprwm/${PN^}.git"
+	EGIT_BRANCH="main"
+	inherit git-r3
+	S="${WORKDIR}/${PN}-${PV}"
```

```suggestion
    inherit git-r3
	EGIT_REPO_URI="https://github.com/hyprwm/${PN^}.git"
```

Setting S shouldn't be necessary as git-r3 should set the correct value. And EGIT_BRANCH should be correct as hyprlands HEAD is main already.